pkgs/development/python-modules/redshift-connector/default.nix

```
@@ -0,0 +1,58 @@
+{ beautifulsoup4
+, boto3
+, buildPythonPackage
+, fetchFromGitHub
+, lib
+, lxml
+, packaging
+, pytest
+, pytest-cov
+, pytest-mock
+, pytestCheckHook
+, pythonOlder
+, pytz
+, requests
+, scramp
+}:
+
+buildPythonPackage rec {
+  pname = "redshift-connector";
+  version = "2.0.910";
```

```suggestion
  version = "2.0.910";
  format = "setuptools";
```

Set an explicit format for python builds.